Lib/test/test_msvcrt.py

```diff
@@ -134,7 +136,10 @@ def test_set_error_mode(self):
         returned = msvcrt.set_error_mode(msvcrt.OUT_TO_STDERR)
         self.assertIs(type(returned), int)
         self.assertNotEqual(returned, -1)
-        self.assertEqual(old, returned)
+
+        returned = msvcrt.set_error_mode(msvcrt.REPORT_ERRMODE)
+        self.assertIs(type(returned), int)
```

Comment:
Type checks are mostly redundant if we check for known value. It is enough to check the type only once, after the first call, as we expect that it is the same for all calls.